### DISCUSSION TITLE
3D Weather Sandbox

### DISCUSSION THREAD

**Discussion by Weathermandavid** [niels747/2D-Weather-Sandbox#63] [Ideas]
Just like the 2D Weather sandbox, a huge upgrade to a 3D version with 3D clouds updrafts, wind currents, lakes, oceans, terrain, building, trees and, more would be a very good idea. I don't mind the 2D version but it's always fun to go a step further and create a 3D version.

**Comment by wilburcoding** [comment]
would be very cpu intensive

**Reply by FalloutDEV** [reply]
get a good pc lol

**Comment by odvigajlo** [comment]
This **is possible** i mean i already heard of 3d weather sim that could be run on laptop! it would be less interactive at this point due to perfomance but this **is possible**!

**Comment by Weathermandavid** [comment]
I agree. I always like 3D simulations and I think it would be more fun,
that's just my personal opinion since I use the 2D Weather Sandbox every
single day as I'm a weather enthusiast.

**Comment by niels747** [comment]
I dont know of a way to make it run fast enough without a supercomputer. I'm just gonna stick with 2D.

**Comment by Weathermandavid** [comment]
On Fri, Feb 10, 2023 at 2:42 AM Niels Daemen ***@***.***>
wrote:

> I dont know of a way to make it run fast enough without a supercomputer.
> I'm just gonna stick with 2D.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/niels747/2D-Weather-Sandbox/discussions/63#discussioncomment-4933531>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A5J42PRWMV42PRZTKWA2KJTWWYLPTANCNFSM6AAAAAAUI7HGYM>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>
So that means there’s no way to make it a 3D version?

**Comment by CheetahCyberBoi** [comment]
It's possible, it would just require a NASA supercomputer to run at more than -1 fps.

**Reply by coolkid0281** [reply]
Yeah

**Comment by niels747** [comment]
Yea, so maybe in 20 years...
